ops/ccs-ops-misc/load_test/runtests.py

```diff
@@ -11,6 +11,8 @@
 from common import config, test_setup as setup
 from locust.main import main
 
+from common.stats import StatsEnvironment, StatsStorageConfig
```

Comment:
Is `StatsEnvironment` used here?

Reply:
Good catch! A previous iteration of these changes had multiple command-line arguments with one specifying the environment and the code in `runtests.py` handling that argument used `StatsEnvironment`.

I'll remove the unused import.